IFC4x3/PropertyConstants/f/FOUR_ASPECT_CAB_SIGNAL_CONNECTION_SIGN_0TvnmygdjFyuZ1Bn7d8Wbd.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="FOUR_ASPECT_CAB_SIGNAL_CONNECTION_SIGN_0TvnmygdjFyuZ1Bn7d8Wbd" Name="FOUR_ASPECT_CAB_SIGNAL_CONNECTION_SIGN" UniqueId="1de71c3c-aa7b-4ff3-88c1-2f11e7220967" />
```

Comment:
Would be consistent if `FOUR_ASPECT_CAB_SIGNAL_CONNECT_SIGN`

Reply:
Done. Commit : 426664c10f4b7a9f9b1eff4dddaa4ee62a65d0e3